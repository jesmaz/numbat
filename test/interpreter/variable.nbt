var a : 10
a = 90
ref b : a
var c : a
a = 6
b
c