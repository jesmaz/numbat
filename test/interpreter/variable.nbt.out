10
90
90
90
6
6
90